data_structures/stacks/Trapping_Rain_Water.py

```diff
@@ -0,0 +1,52 @@
+"""
+	Every element represents the height of the block. The width of the block can be considered 1.
+	Find the volume of trapped rainwater between these block.
+"""
+
+height = [34, 2, 5, 2, 3, 23, 18, 23, 45]
+answer = 162
+
+
+def max_water(height):
+
+    """
+    Get the maximum unit of water trapped between the blocks
+
+    >>> max_water(height) == answer
```

Comment:
You need to create an actual variable for `height` and `answer`. This raised an error, hence why the checks failed